themes/bootstrap3/templates/librarycards/editcard.phtml

```diff
@@ -2,6 +2,8 @@
   // Set up page title:
   $pageTitle = empty($this->card->id) ? 'Add a Library Card' : 'Edit Library Card';
   $this->headTitle($this->translate($pageTitle));
+  // If the card is managed by Shibboleth, enable only renaming the card
+  $rename = ($this->loginMethod == 'shibboleth' || $this->shibboleth);
```

Comment:
As far as I can see, `$this->shibboleth` check would break normal library card handling if the user logs in using Shibboleth. You should still be able to handle normal library cards regardless of the login method.

Reply:
I'd also rename $rename to $renameOnly.

Reply:
It is too specific for our use case so I decided to enable editation of library cards when shibboleth is used for authentication.